MainWindow.xaml.cs

```diff
@@ -161,6 +161,23 @@ private void OpenData_Click(object sender, RoutedEventArgs e)
             }
         }
 
+
+        private void ImportData_Click(object sender, RoutedEventArgs e)
+        {
+            var dialog = new System.Windows.Forms.OpenFileDialog();
+            dialog.Filter = "JSON Files (*.json)|*.json";
+            System.Windows.Forms.DialogResult result = dialog.ShowDialog();
+            if (result == System.Windows.Forms.DialogResult.OK)
+            {
+                string input = File.ReadAllText(dialog.FileName);
+                Data.DynamicInputPack inputPack = new Data.DynamicInputPack();
+                inputPack.InportfromLocation(dialog.FileName);
```

Comment:
This function should likely deal with errors in some way as well.  The most basic error we need to handle is if any of the nullable objects don't have a value when you call `.Value` they will throw.

Reply:
unfortunately, i do not know which values you mean exactly?

Reply:
Sorry, I wasn't very clear.  For instance in this code:

```
Region.X = Reader.ReadAsDouble().Value;
```

The `ReadAsDouble` returns a nullable.  If the value read isn't a double (I think) or doesn't exist, then it will throw.

Considering these are generated, it's probably **very** unlikely but wrapping this in a try/catch would alleviate this issue.

Reply:
I only use when I know what type they have, so the pack must be faulty for it to render zero, but I can test how it runs with faulty pack, so we would have a kind of recovery function.

Reply:
I have created the bassis improt for v2 packs, but currently they will become v1 packs again, but so we can easily add the new functions when they are available in the tool.

The import function is now very stable, just like the opening function.